packages/deploy/deploy/1000_landsale/1000_deploy_estate_sale.ts

```diff
@@ -0,0 +1,139 @@
+import {Deployment} from 'hardhat-deploy/dist/types';
+import {DeployFunction} from 'hardhat-deploy/types';
```

Comment:
Dont import from dist

```suggestion
import {DeployFunction, Deployment} from 'hardhat-deploy/types';
```